src/resolvelib/reporters.py

```diff
@@ -30,8 +30,9 @@ def adding_requirement(self, requirement, parent):
             requirements passed in from ``Resolver.resolve()``.
         """
 
-    def start_backtracking(self, causes):
-        """Called when backtracking started because of collision in requirements
+    def resolving_conflicts(self, causes):
+        """Called when a collision in requirements was detected and
+        an attempt to resolve them was started
```

Comment:
```suggestion
    def resolving_conflicts(self, causes):
        """Called when starting to attempt requirement conflict resolution.
```

Let's keep this succinct.